### ISSUE TITLE
what is normalize in imagenet_eval.py 101

### ISSUE THREAD

**Issue by jiahong-chen** [Cadene/pretrained-models.pytorch#173]
My editor say Undefined variable 'normalize', so i want to know where normalize is defined ?

**Comment by DKandrew** [comment]
That file `imagenet_eval.py` needs some maintenance. You should change it to `transforms.Normalize(mean=model.mean, std=model.std),`
